core/metacling/src/TClingCallbacks.cxx

```diff
@@ -919,7 +922,48 @@ void TClingCallbacks::UnlockCompilationDuringUserCodeExecution(void *StateInfo)
    TCling__UnlockCompilationDuringUserCodeExecution(StateInfo);
 }
 
-static bool shouldIgnore(llvm::StringRef FileName) {
+static bool shouldIgnore(const std::string& FileName,
+                         cling::DynamicLibraryManager* dyLibManager) {
```

Comment:
In that area of code, what is the convention for always-non-zero object parameter (pointer or ref)?

Reply:
We can do either way. I do not think there is a convention.

Reply:
I tend to prefer the reference to avoid the 'temptation' to add a ```if (!param) return```